File: swagger/models/get_memberships_response.py
from .base_model import BaseModel

from .membership import Membership

class GetMembershipsResponse(BaseModel):
	"""
	no description available

	Attributes:

		memberships
		Details about the memberships.
	"""

	input_map = {
		'Memberships': ['memberships', Membership],
		}

	output_map = {
		'memberships': 'Memberships',
		}

	memberships = None

File: swagger/Models/GetMembershipsResponse.php
<?php

namespace App\Mindbody\Models;

/**
 * no description available
 */
class GetMembershipsResponse extends BaseModel
{
	/**
	 * Details about the memberships.
	 * 
	 * @var Membership[] 
	 */
	public $memberships;

	protected function getInputMap()
	{
		return [
			'Memberships' => ['memberships', 'array', Membership::class],
			];
	}

	protected function getOutputMap()
	{
		return [
			'memberships' => ['Memberships', 'array', Membership::class],
			];
	}
}
